Tails specific: we start NetworkManager ourselves, via tails-greeter, at
PostLogin time.

--- chroot.orig/etc/init.d/network-manager	2012-09-24 10:05:13.157051525 +0200
+++ chroot/etc/init.d/network-manager	2012-09-24 10:03:30.638108333 +0200
@@ -8,2 +8,2 @@
-# Default-Start:     2 3 4 5
+# Default-Start:
-# Default-Stop:      0 1 6
+# Default-Stop:      0 1 6 2 3 4 5
